### DISCUSSION TITLE
Just wanted to say thanks

### DISCUSSION THREAD

**Discussion by Scbri** [aqualinkd/AqualinkD#250] [General]
After months of over researching and downloading my jandy rs wiring diagrams. it only took 20 minutes from time i finally wired the usb2rs485 adapter into my rs-6 board and connected to pi zero w to get my spa only system working perfectly from the built in web ui on a ipad mini.  Every feature, sensor and control worked on the first try.

So many thanks for all your work to provide us with such a well documented installed pathway.  Couldn't believe my pool guy telling me i had to upgrade my entire board to a new version (I am on Rev Q!!!!! one version to early)and then i would have to buy the app/network connected pool system. $2000 for crap.

cant thank you enough.  Now I have smack my pool quy around some more since he said my system doesn't support VSP's

**Comment by zullot** [comment]
Could you possibly post a picture of your wiring setup?

I've been reading and re-reading the install wiki over and over and I feel like I'm missing something. I can't get the usb test command to output any actual words...

**Reply by Scbri** [reply]
I'll take a picture and post as soon as i get a chance

**Reply by sfeakes** [reply]
Depending on what’s connected to your RS485 bus you may not get any words. If you get some output, then try serial-logger, if that doesn’t pick up anything then reverse the data + & - wires and try again.

the important thing is getting some data / output with the usb test command.
